src/cogniteClient.ts

```diff
@@ -286,6 +291,11 @@ export default class CogniteClient {
       return didAuthenticate ? retry() : reject();
     });
 
+    const { accessToken } = options;
```

Comment:
please verify that `accessToken` contains same `project` as passed `project`

Reply:
How can we verify this??

Reply:
And why do we need this check? Which scenario will that be?

Reply:
The case is:
- You login to `akerbp` but provide access token for `publicdata`
- When you then call `client.login.status()` it will give you the status for the wrong project.
 That is confusing.

Any solutions @polomani ?

Reply:
how will all other endpoints behave? If they return 401 then we can just to fix the `login.status()` to return null